src/utils.ts

```diff
@@ -0,0 +1,9 @@
+/** if the browser supports WebGL */
+export const isWebGLAvailable = () => {
+	try {
+		const canvas = document.createElement("canvas");
```

Comment:
please delete the canvas
![image](https://github.com/bs-community/skinview3d/assets/31113245/6efc52b5-3a1b-443d-b76a-485b7020a783)